### PR TITLE
Merge the Desktop and Mobile versions of the codebase (Ax 344 0.3 on Jira)

### PR DESCRIPTION
**Pull request by mretana1999**
Started to make UI responsive to mobile screen sizes:

- Implemented top navbar on V1App.
- Implemented bottom navbar on V1App.
- Made DesktopFarm responsive and simplified code (some styling details still need to be fixed)
- Fixed some issues with wallet connectivity in Controller.dart
- Added comments in Controller.dart and Dialog.dart that indicate which lines need to be replaced so that the app runs on an Android device.
- Deleted flutter svg package which was not being used.

### CODE REVIEW

File: lib/service/AthleteApi.dart [hunk 1]
```diff
@@ -4,6 +4,9 @@ import 'dart:convert';
 import 'package:http/http.dart' as http;
 
 class AthleteApi {
+  static final String _baseURL = 'https://db.athletex.io';
+  // static final String _BaseURL = 'http://139.99.74.201:8080';
```
Comment (ketonkss4) [inline lib/service/AthleteApi.dart:8]: Just delete this comment it is not needed anymore

Comment (ketonkss4) [inline lib/service/AthleteApi.dart:8]: Just delete this comment it is not needed anymore

File: lib/service/AthleteApi.dart [hunk 2]
```diff
@@ -37,4 +40,27 @@ class AthleteApi {
     print(athletesList);
     return athletesList;
   }
+
+  static Future<List<Athlete>> getAthletesFromIdsDict(
+      BuildContext context) async {
+    Map<String, List<int>> athleteIdDict = {
+      "ids": [9038, 22564, 21693, 18882],
```
Comment (ketonkss4) [inline lib/service/AthleteApi.dart:47]: These are magic numbers if we need these as constants make them 
static final in the class and give them a meaningful name

File: lib/service/AthleteApi.dart [hunk 3]
```diff
@@ -37,4 +40,27 @@ class AthleteApi {
...
+        headers: {"Content-Type": "application/json"}, body: jsonRequestBody);
+    print(athleteResponse.statusCode);
+
+    if (athleteResponse.statusCode == 200) {
+      var athleteResponseList = jsonDecode(athleteResponse.body) as List;
+      return athleteResponseList
+          .map((athlete) => Athlete.fromJson(athlete))
+          .toList();
+    } else {
+      throw Exception("Failed to load athlete");
```
Comment (ketonkss4) [inline lib/service/AthleteApi.dart:63]: Does this just crash the app or it is handled somewhere else

**Review by ketonkss4 — CHANGES_REQUESTED** [review]

File: lib/service/AthleteApi.dart (hunk 1 above)
Comment (pearlson16) [reply lib/service/AthleteApi.dart:8]: Completed

File: lib/service/AthleteApi.dart (hunk 2 above)
Comment (pearlson16) [reply lib/service/AthleteApi.dart:47]: complete

File: lib/service/AthleteApi.dart (hunk 3 above)
Comment (mretana1999) [reply lib/service/AthleteApi.dart:70]: New commit now handles API errors in LandingPage by showing a "Reload Application" button.

**Review by ketonkss4 — APPROVED** [review]

**Review by user2745 — APPROVED** [review]

**Pull request merged**